docs/book/src/component-config-tutorial/testdata/project/test/e2e/e2e_suite_test.go

```diff
@@ -20,7 +20,9 @@ import (
 	"fmt"
 	"testing"
 
+	// nolint:revive
```

Comment:
Rather than having all of these `nolint:revive` lines, I think we are safe to assume dot imports will be used for intentional reasons. I think it would be cleaner to just remove the `dot-imports` rule from line 28 in `.golangci.yml`.

Reply:
Agreed. One way maybe:
```yaml
[rule.dot-imports]
  arguments = [{ allowedPackages = ["github.com/onsi/ginkgo/v2","github.com/onsi/gomega"] }]
```

Reply:
We should not use dot imports
Just the case of ginkgo and gomega which are an exception.
So, we still want the check 

If a config like 

```
[rule.dot-imports]
  arguments = [{ allowedPackages = ["github.com/onsi/ginkgo/v2","github.com/onsi/gomega"] }]
```
Works then, It is totally fine.